### PR TITLE
build: support building ccm for multi-platform

### PR DESCRIPTION
**Pull request by luncj**
#### What this PR does / why we need it:

Support building `cloud-controller-manager` for `arm` / `arm64` / etc platforms.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by andyzhangx — COMMENTED** [review]
/ok-to-test

File: Makefile [hunk 1]
```diff
@@ -115,14 +115,19 @@ docker-pull-prerequisites: ## Pull prerequisite images.
 
 buildx-setup:
 	docker buildx inspect img-builder > /dev/null || docker buildx create --name img-builder --use
-	# enable qemu for arm64 build
-	# https://github.com/docker/buildx/issues/464#issuecomment-741507760
-	docker run --privileged --rm tonistiigi/binfmt --uninstall qemu-aarch64
```
Comment (andyzhangx) [inline Makefile:120]: we don't need this workaround any more?

Comment (luncj) [reply Makefile]: Nah, my bad. fixed.

**Pull request merged**